pkg/internal/checksum/checksum.go

```diff
@@ -0,0 +1,60 @@
+package checksum
```

Comment:
Seems like this pkg is only used by the api, would it be a good fit to api/itnernal?

Reply:
done.